src/nmodl/nocpout.cpp

```diff
@@ -704,24 +724,31 @@ extern Memb_func* memb_func;\n\
     /*declaration of the range variables names to HOC */
     Lappendstr(
         defs_list,
-        "static void nrn_alloc(Prop*);\nstatic void  nrn_init(NrnThread*, Memb_list*, int);\nstatic void nrn_state(NrnThread*, Memb_list*, int);\n\
-");
+        "static void nrn_alloc(Prop*);\n"
+        "static void nrn_init(_nrn_model_sorted_token const&, NrnThread*, Memb_list*, int);\n"
+        "static void nrn_state(_nrn_model_sorted_token const&, NrnThread*, Memb_list*, int);\n");
     if (brkpnt_exists) {
-        Lappendstr(defs_list,
-                   "static void nrn_cur(NrnThread*, Memb_list*, int);\nstatic void  "
-                   "nrn_jacob(NrnThread*, Memb_list*, int);\n");
+        Lappendstr(
+            defs_list,
+            "static void nrn_cur(_nrn_model_sorted_token const&, NrnThread*, Memb_list*, int);\n"
+            "static void nrn_jacob(_nrn_model_sorted_token const&, NrnThread*, "
+            "Memb_list*, int);\n");
     }
     /* count the number of pointers needed */
     ppvar_cnt = ioncount + diamdec + pointercount + areadec;
     if (net_send_seen_) {
         tqitem_index = ppvar_cnt;
-        ppvar_semantics(ppvar_cnt, "netsend");
+        ppvar_semantics(
+            ppvar_cnt,
+            "netsend",
+            "_tqitem",
+            "void*" /* TQItem* really, but that's not defined in translated MOD file code */);
         ppvar_cnt++;
     }
     if (watch_seen_) {
         watch_index = ppvar_cnt;
         for (i = 0; i < watch_seen_; ++i) {
-            ppvar_semantics(i + ppvar_cnt, "watch");
+            ppvar_semantics(i + ppvar_cnt, "watch", "_watch_array", "void*" /* FIXME */);
```

Comment:
> /* FIXME */

`nitpick`: couldn't guess what would be that fix.

Reply:
Fair comment...

Reply:
Changed in 1e4916d9e.